### DISCUSSION TITLE
Stopping a Kafka cluster without deleting the instances

### DISCUSSION THREAD

**Discussion by viopinto** [strimzi/strimzi-kafka-operator#4593] [Q&A]
Hi Team,

Is there a way to retain my Kafka cluster without having it to serve any requests ? Basically we need a maintenance mode, wherein for a given point in time, we do not want our Kafka cluster to be serving or managing any requests. Trying to figure out if there is something similar to stopping a Kafka Cluster without deleting it.

Version : Strimzi 0.20

**Comment by scholzj** [comment]
In 0.20 you can scale down the operator pod and then you can do with the pod whatever you want. And when you are finished, you can just scale the operator back to one replica. 

From 0.22.0 you can also pause the reconciliation for a particular Kafka custom resource -> so you can just pause it, do your maintenance and unpause it again without scaling the operator up and down: https://strimzi.io/docs/operators/latest/full/using.html#proc-pausing-reconciliation-str

**Reply by viopinto** [reply]
Scaling down the operator pod does work in one scenario. In addition, I have another scenario of multiple Kafka clusters being deployed under the hood of a single cluster operator.  Is there any way that I could have one of the Kafka cluster not serving any requests without hindering the other Kafka cluster?

**Reply by scholzj** [reply]
The pausing of reconciliation I pointed to above is per cluster. So that should work for you, or?

**Reply by viopinto** [reply]
Thanks for your prompt reply.  I have not tried pausing the reconciliation, as I am currently on Strimzi 0.20 and was trying to find any alternative in the respective version.

**Reply by scholzj** [reply]
I'm afraid I do not have anything for 0.20.
